ide-common/src/main/kotlin/org/digma/intellij/plugin/posthog/ActivityMonitor.kt

```diff
@@ -195,6 +204,20 @@ class ActivityMonitor(private val project: Project) /*: Runnable, Disposable*/ {
         )
     }
 
+   fun registerSpanLinkClicked(panelName: String, source: String? = null) {
```

Comment:
```suggestion
   fun registerSpanLinkClicked(panelName: PanelName, source: String? = null) {
```

maybe lets use `PanelName` enum, so values would be enclosed ,instead of arbitrary strings

Reply:
added